06_modules_lab/03.py

```diff
@@ -1,11 +1,22 @@
-""" Write a program that searches current working directory
-for files larger than 1MB. Every time you find such a file print
-its name to the user.
-
-- When the program finds a large file. It should ask the user
-  a message asking if she wants to delete it, and delete the
-  file if requested
-
-- Take threshold and path as command line arguments
-"""
-
+
+#Search for files that their size is bigger than 1mb and print the name of the file
+
+import os, sys
+
+if len(sys.argv) != 3:
+    print "Didn't use properly"
+    sys.exit()
+
+search_dir = sys.argv[1]
+min_file_size = int(sys.argv[2])
+
+for root,dirs,files in os.walk(search_dir):
+    for name in files:
+        file_path = os.path.join(root, name)
+        file_size = os.path.getsize(file_path)
+        if file_size >= min_file_size:
+            print "{0} size is {1}. Do you want to delete this ? [y/n] ".format(file_path, file_size)
+            answer = raw_input()
+            if answer == "y" or "yes":
+                print os.remove(file_path)
+

```

Comment:
This last pasted not is a script with a syntax error though maybe it is only because how it was pasted here.
```
 File "a.py", line 3
    print "Please enter 2 numbers !"
        ^
IndentationError: expected an indented block
```